test/basics.fs

```diff
@@ -59,10 +59,26 @@ let addEight = add 3 << addFive
 let strLen (s: string) = s.Length
 let strLen2 s = String.length s
 let listLen (s: List<_>) = s.Length
+let strConcat(sep: string, strList:List<string>) = String.concat sep strList
+let hasAs str = String.exists (fun x -> x = 'A') str
+let allAs str = String.forall (fun x -> x = 'A') str
+let times8 str = String.replicate 8 str
+
+let strToAs str = String.map (fun c -> 'A') str
+
+let removeFirstTwo str = String.mapi (fun i c -> if i <= 1 then '_' else c) str
+let removeAfterTwo str = String.mapi (fun i c -> if i >= 2 then '_' else c) str
+
+let repeatIndexTimes n str = String.init n (fun i -> String.replicate i str)
+
+// need System.Char
```

Comment:
It might be ok to provide erlang implementations of a small set of System static functions in core type namespaces.